src/Runner/TFLiteRunner.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PythonShell} from 'python-shell';
+
+import {Logger} from '../Utils/Logger';
+import {Runner} from './Runner';
+
+
+export class TFLiteModelRunner extends Runner {
+  constructor(l: Logger) {
+    super(l, 'tflite');
+    console.log('tflite model runner constructor')
+  }
+
+  runModel(): void {
+    super.runModel();
+    console.log('TFLiteRunner.runModel()');
+
+    let options = {
+      pythonPath: '/usr/share/one/bin/venv/bin/python',  // TODO: change to docker's
+      pythonOptions: ['-u'],                             // get print results in real-time
+      scriptPath: __dirname,                             // Locates with this file
+      args: [this.modelPath]
+    };
+
+    PythonShell.run('execTflite.py', options, function(err, results) {
```

Comment:
1. Can you use just `shell` instead of `PythonShell`? (Just curious.)
2. `execTflite.py` would be in ONE? or ONE-vscode?

Reply:
> * Can you use just `shell` instead of `PythonShell`? (Just curious.)

I applied `PythonShell` because it helps to specify the python path.
But, since we are aiming to include docker container for execution environment, `shell` could be better to run docker bash script.. 



> * `execTflite.py` would be in ONE? or ONE-vscode?

In this case maybe only for ONE-vscode.

Reply:
I'm not sure `In this case maybe only for ONE-vscode.`. If it is in ONE, users who want to run it are easy for running it and `copyFiles` also is unnecessary. Any reason to place it in ONE-vscode?

Reply:
Ah, my word was...
- ONE's toolchains will be supported by docker
- If the `execTflite.py` is supported by ONE, ONE-vscode would use the `execTflite.py` smoothly.

Reply:
Introducing `execTflite.py` to ONE can satisfy both `ONE` toolchain user and `ONE-vscode` users if they are trying to run value test. 
And, I felt your suggestion is related with https://github.com/Samsung/ONE/issues/8743. Let me and our team think about it more carefully. :smile:

Reply:
@yunjayh,
How about creating an issue in ONE repo to discuss this issue?